core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -416,8 +433,10 @@ private void printHelp () {
             System.out.println("fire - will fire the gun if all conditions for firing it are met.");
             System.out.println("inv - will display all items in current player's inventory.");
             System.out.println("info - will display current player's heat level, number of moves left and the number of gun parts collected by the team.");
+            System.out.println("exit - Will exit the game.");
+            System.out.println("map - will show the map to the user.");
         }
-
+        //Loads all inputs and returns them as a List
         public ArrayList<String> loadInputs (String path) throws FileNotFoundException {
```

Comment:
we changed a bit this function. Inside the ArrayList, we should have another Array of string String[]